Earthfile

```diff
@@ -82,27 +55,3 @@ test-bootable:
     ARG CREATE_VM=true
     RUN date
     RUN go run github.com/onsi/ginkgo/v2/ginkgo run --label-filter "bootable" -v --fail-fast -r ./e2e
-
-last-commit-packages:
-    FROM quay.io/skopeo/stable
-    RUN dnf install -y jq
-    WORKDIR build
-    ENV jqQuery='.Tags | map(select(. | contains("-repository.yaml"))) | sort_by(. | sub("v";"") | sub("-repository.yaml";"") | sub("-git.*";"") | .[0:12] | tonumber) | .[-1]'
-    RUN skopeo list-tags docker://quay.io/kairos/packages | jq -rc "${jqQuery}" >  REPO_AMD64
-    RUN skopeo list-tags docker://quay.io/kairos/packages-arm64 | jq -rc "${jqQuery}" > REPO_ARM64
-    SAVE ARTIFACT REPO_AMD64 REPO_AMD64
-    SAVE ARTIFACT REPO_ARM64 REPO_ARM64
-
-bump-repositories:
```

Comment:
How do we bump the luet repository yamls without this?

Reply:
Renovate should take care of it as we do on other places. This was put in place when we could not make it renovate do it automatically but that has been fixed.